SDKAndroid/src/main/java/app/cybrid/sdkandroid/core/BigDecimal.kt

```diff
@@ -43,8 +43,26 @@ class BigDecimal(internal val value:JavaBigDecimal) : Comparable<BigDecimal> {
         return BigDecimal(value.pow(n))
     }
 
+    fun toInt(): Int {
+
+        val intValue = value.toInt()
+        if (intValue < 0) {
```

Comment:
Can we compare `intValue == value.toInt()`?

Same below for double. That would be a better test.

Reply:
Done, we compare with intValueExact, that allows to have the last correct number before Arithmetic throw raise!!

`intValue == value.intValueExact()`